packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ Changelog
 
 Unreleased
 ----------
-
+### Changed
+* Replaced local key code value with keycode-js
```

Comment:
nit: can you add a blank line after this for readability

Reply:
https://github.com/cerner/terra-clinical/pull/479/commits/09ba9949452822abb1e3446957588da693d98182#diff-7a1a433761e6faaf0aef62989abdc463L6

Reply:
Looks like there might have been a merge conflict, this needs to be moved back to the unreleased section.

Reply:
@jmsv6d is right. Sorry @anjaliroy19, I didn't catch this when we were fixing the conflicts